### PR TITLE
Changes defib sprites

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds Nixons defib sprites. A marine one and a civilian one.

## Why It's Good For The Game
more variety. Fits the style better

## Changelog
:cl: Nixon
add: Added a civil defib and marine defib
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -26. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by psykzz** [comment]
On the line `:cl:` if you add Nixon you can add them to the change log

for example  
```
## Changelog
:cl: Nixon
add: Added a civil defib and marine defib
/:cl:```

File: code/game/objects/items/defibrillator.dm [hunk 1]
```diff
@@ -72,8 +72,8 @@
 
 	defib_cooldown = world.time + 2 SECONDS
 	ready = !ready
-	user.visible_message("<span class='notice'>[user] turns [src] [ready? "on and takes the paddles out" : "off and puts the paddles back in"].</span>",
-	"<span class='notice'>You turn [src] [ready? "on and take the paddles out" : "off and put the paddles back in"].</span>")
+	user.visible_message("<span class='notice'>[user] turns [src] [ready? "on and opens the cover" : "off and puts the paddles back in"].</span>",
```
Comment (XSlayer300) [inline code/game/objects/items/defibrillator.dm:75]:
```suggestion
	user.visible_message("<span class='notice'>[user] turns [src] [ready? "on and opens the cover" : "off and closes the cover"].</span>",
```

Comment (Hughgent) [reply code/game/objects/items/defibrillator.dm:75]: Please add a space after `ready` and before the `?`

**Comment by jroinc** [comment]
i like the green ones

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-32). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**